index.html

```diff
@@ -76,7 +76,7 @@
   </head>
   <body>
     <section id="abstract">
-      <p>This specification does neat stuff.</p>
+      <p>This specification defines a JSON Web Token (JWT) [RFC7519] representation of Verifiable Credentials.</p>
```

Comment:
similar to the comment on another PR, I think it should be `This specification defines how Verifiable Credentials can be secured using JSON Web Token (JWT) [RFC7519]`. Though JWS might be more appropriate in this sentence structure...

Reply:
I like @Sakurann's rephrasing better as a starting point. I might suggest that both JWS and JWT should be mentioned, as I believe they travel together in this scenario, so maybe something like, `This specification defines how Verifiable Credentials can be represented as JSON Web Tokens (JWT) [RFC7519] and secured using JSON Web Signatures (JWS) [RFC7515].`